yew-macro/tests/macro/html-shorthand-prop-pass.rs

```diff
@@ -0,0 +1,20 @@
+use yew::prelude::*;
+
+fn compile_pass() {
+    let placeholder = "placeholder-text";
+    let maxlength = "20";
+    let href = "https://yew.rs/docs/";
+    let style = "background-color:blue";
+    let size = 25;
+
+    html! { <input placeholder= />};
```

Comment:
Hmm, I thought this would look ok but after some time has passed and looking again... it just looks like someone forgot to set the prop value here. I initially was in favour of adding this functionality behind a feature flag but now I'm not even sure of that. I think we should revisit the syntax

Reply:
I personally don't have a strong preference for this syntax. The feature request was a target of opportunity for me to dig into Yew and contribute to the project. I don't find it very cumbersome to explicitly specify prop names. However, if the original requester of the feature still really wants it I'm happy to help contribute to ideas for alternate syntax.

If there is no one championing this feature though, then we can just close the pull request unmerged.